.github/workflows/noir-js.yml

```diff
@@ -14,11 +14,39 @@ jobs:
       - name: Install dependencies
         run: yarn install --immutable
 
+      - name: Install jq
+        run: sudo apt-get install jq
+
+      - name: Install cargo and wasm-bindgen
+        run: |
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+          source $HOME/.cargo/env
+          cargo install -f wasm-bindgen-cli --version 0.2.86
+
+      - name: Install toml2json
+        run: |
+          source $HOME/.cargo/env
+          cargo install toml2json
+
+      - name: Install wasm-opt
+        run: |
+          npm i wasm-opt -g
+
+      - name: Install wasm32-unknown-unknwown target
+        run: |
+          rustup target add wasm32-unknown-unknown
+
+      - name: Build noirc_abi
```

Comment:
Seems we need to build noirc_abi since its in the workspace, before we can build noir_js as its dependent on this